content/en/docs/Developers/dev-environment.md

```diff
@@ -13,6 +13,9 @@ If you are stuck or have any questions, please join our [Discord server](https:/
 give us a shout on the `#dev` channel
 {{% /pageinfo %}}
 
+### Docs Contributions
```

Comment:
Not sure if this is the best place to add this information. Do you really think it is necessary, given that all pages have links to the documentation project:
<img width="353" alt="Screen Shot 2020-10-05 at 7 38 31 PM" src="https://user-images.githubusercontent.com/331353/95142769-78272780-0742-11eb-893d-0cf446768a8d.png">

Reply:
Fair point! I went ahead and pulled that change.

Let me know if there's anything else.

Thanks

Reply:
LGTM